djlsp/parser.py

```diff
@@ -208,10 +208,11 @@ def get_block_completions(self, match: Match, **kwargs):
         prefix = match.group(1).strip()
         logger.debug(f"Find block matches for: {prefix}")
         block_names = []
-        if "/templates/" in self.document.path:
-            template_name = self.document.path.split("/templates/", 1)[1]
-            if template := self.workspace_index.templates.get(template_name):
-                block_names = self._recursive_block_names(template.extends)
+        re_extends = re.compile(r""".*{% ?extends ['"](.*)['"] ?%}.*""")
+        for line in self.document.lines:
+            if matches := re_extends.match(line):
```

Comment:
Good idea, maybe use `re.search` and `self.document.source`

Reply:
Good idea, was not aware, that `self.document.source` exists. Have updated it now.